CMakeLists.txt

```diff
@@ -119,7 +121,19 @@ add_subdirectory(cmake)
 list(APPEND CMAKE_MODULE_PATH "${PROJECT_SOURCE_DIR}/cmake")
 
 # Find necessary packages
-find_package(Compiler REQUIRED)
+if (ONEMKL_SYCL_IMPLEMENTATION STREQUAL "hipSYCL")
+  message(STATUS "Looking for hipSYCL")
+  find_package(hipSYCL CONFIG PATHS /opt/hipSYCL/lib/cmake/ REQUIRED)
```

Comment:
I think we can remove the `PATHS` specification to make it work better with non-system-wide installations. (In any case, I think it should be `lib/cmake/hipSYCL`, not `lib/cmake`)